packages/bar/src/props.js

```diff
@@ -34,6 +34,8 @@ export const BarPropTypes = {
     layout: PropTypes.oneOf(['horizontal', 'vertical']).isRequired,
     reverse: PropTypes.bool.isRequired,
     valueScale: scalePropType.isRequired,
+    indexScale: scalePropType.isRequired,
```

Comment:
Actually, we should use `bandScalePropTypes` as other scale types aren't supported.